test/utils/runners.go

```diff
@@ -993,7 +993,6 @@ func MakePodSpec() v1.PodSpec {
 }
 
 func makeCreatePod(client clientset.Interface, namespace string, podTemplate *v1.Pod) error {
-	var err error
```

Comment:
doesn't removing this cause a compile error on line 1003?

Reply:
maybe not ,but if want left the line, i think line 997 to change like this :
if _, err = client.Core().Pods(namespace).Create(podTemplate); err == nil {
like 900 line,what about you? @liggitt  thanks

Reply:
Yes, put this back as it was and change line 900